draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -179,6 +178,13 @@ Dynamically generated names can be used to track users if used too often.
 Conversely, registering too many names will also generate useless processing.
 The proposed rule is to create and register a new generated name for a given IP address on a per execution context.
 
+Hostnames resolving to multiple IP adresses
```

Comment:
This section does not really fit in the "Privacy Guidelines" section.
I would keep these changes in section "ICE Candidate Processing".